src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1323,6 +1328,26 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly genesetMolecularProfile = remoteData<MolecularProfile | undefined>({
+        await: () => [
+            this.molecularProfilesInStudies
+        ],
+        invoke: () => {
+            const GENESET_SCORE = "GENESET_SCORE";
```

Comment:
add this to `AlterationTypeConstants` in this file

Reply:
Ah, I see, that was added last week, before I wrote this code. Yes, that does sound like a sensible place for it 👍